api/src/pcapi/routes/auth/blueprint.py

```diff
@@ -0,0 +1,24 @@
+from flask import Blueprint
+from flask_cors import CORS
+
+from pcapi import settings
+from pcapi.routes.auth import utils
+from pcapi.serialization.spec_tree import ExtendedSpecTree
+from pcapi.serialization.utils import before_handler
+
+
+auth_blueprint = Blueprint("auth", __name__, template_folder="templates")
+CORS(
+    auth_blueprint,
+    origins=settings.CORS_ALLOWED_ORIGINS_NATIVE,
+    supports_credentials=True,
+)
+auth_web = ExtendedSpecTree("flask", MODE="strict", before=before_handler, PATH="/")
+auth_web.register(auth_blueprint)
+
+
+@auth_blueprint.context_processor
+def extra_funcs() -> dict:
+    return {
+        "random_hash": utils.random_hash,
+    }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Je suis pas sur de comprendre à quoi sert ce truc ?‏

Reply:
c'est utilisé pour donner au field un id aléatoire dans templates/components/forms/string_field.html